mjpc/tasks/humanoid/tracking/task.xml

```diff
@@ -1,9 +1,10 @@
 <mujoco model="Humanoid Track">
   <include file="../../common.xml"/>
   <include file="../humanoid.xml" />
-  <size memory="400K"/>
+  <size memory="400K" nuserdata="2"/> <!-- userdata: [motion id, reference time]-->
```

Comment:
you probably don't need nuserdata anymore, right?  maybe also in the quadrotor?

Reply:
Removed from all task xml files now.